lib/install/management/appliance.go

```diff
@@ -536,12 +536,6 @@ func (d *Dispatcher) createAppliance(conf *config.VirtualContainerHostConfigSpec
 		},
 		Dir: "/home/vicadmin",
 	}
-	if settings.HTTPProxy != nil {
```

Comment:
You could just rename the env vars so that golang http client doesn't pick them up and then use them for the connectivity check.